sensor.h

```diff
@@ -7,13 +7,25 @@
  */
 class TemperatureSensor {
 public:
+    virtual void begin();
+
     /**
      * Read the current temperature.
      */
     virtual float temperature() = 0;
+
+    virtual unsigned int elapsed() = 0;
```

Comment:
Please add comments here and below.

Reply:
Fixed in 03d51eeef524e32c61277c2647ff848bb0dc321d